src/styles/index.scss

```diff
@@ -1,3 +1,124 @@
+$main-accent-color: #060b35;
+$secondary-color: #616070;
+$blue-accent-color: #00acdc;
+
 body {
   margin: 0;
+
+  color: $main-accent-color;
+
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+}
+
+.product-card {
+  width: 165px;
+
+  padding: 32px 16px 16px;
+
+  background-color: #fff;
+  color: $main-accent-color;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image-container {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    margin-bottom: 30px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+
+    width: 166px;
+  }
+
+  &__code {
+    color: $secondary-color;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__star {
+    width: 20px;
```

Comment:
I don't believe this is the right size.
And as I said before, you can use the code from Stars' task